File: poc-output/python/divide/src/formula/formula.py
#!/usr/bin/python3

from formula.output import readOutput, removeFile, runFormula, writeOutput


def Run(input1, input2, input3, inputx):
    cmd = f'rit math sum --number_1={input1} --number_2={input2}'
    runFormula(cmd)

    data = readOutput()
    divisor = int(data['output']['sum']) / int(input3)

    print(f'Result: {str(divisor)}')

    removeFile()

    print(inputx)

File: poc-output/node/multiply/src/formula/formula.js
const clc = require('cli-color')
const api = require('./output.js')

function Run(n1, n2, multiplier) {
    api.runFormula(`rit math sum --number_1=${n1} --number_2=${n2}`)
    const data = api.readOutput()
    // const value = parseInt(data.output.sum)

    console.log(data)
    api.writeOutput("vai pocaria")

    // console.log(
    //     clc.blue(
    //         clc.bold('\nResult:'),
    //         value * parseInt(multiplier),
    //         '\n'
    //     )
    // )

    // api.remove()
}

const formula = Run
module.exports = formula
